app/src/main/java/example/erica/kotlinbugdemo/StarRatingView.kt

```diff
@@ -31,9 +31,9 @@ class StarRatingView : LinearLayout {
         // Set up the view
         val dimen = context.resources.getDimensionPixelSize(R.dimen.star_rating_dimen)
         val margin = context.resources.getDimensionPixelSize(R.dimen.star_rating_margin)
-        var i: Int = 0
         // Iterate over the number of stars, adding each one to the layout
-        while (i < numStars) {
+        // Use proper for loop instead of while
+        for (i in 0..numStars) {
```

Comment:
`0 until numStars` here, because `numStars` was not be included in the original loop.

Reply:
Yes, correct, until